workbench/src/Components/LandingPage/LoadDesign/LoadDesign.scss

```
@@ -0,0 +1,197 @@
+                color: #8aa0b8;
+                opacity: 0.8;
+            }
+        }
+        
+        .buttons-right {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            .icon-btn{
+                
+                color:rgb(196, 196, 196);
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)